packages/aws-fargate/images/README.md

```diff
@@ -5,8 +5,8 @@ This folder contains code to build and test the container image that is used by
 
 There are three types of container images, each in their own sub folder:
 
-* `inspector`: A simple Node.js application that inspects its surrounding. It will print environment variables and query the ECS metadata server, and print the resulting HTTP responses. When running this in a Fargate task, it gives valuable insight into what a live Fargate application has available. This has mostly historic value as `@instana/aws-fargate` is pretty feature complete. However, it could become useful again if new AWS Fargate runtime versionsbehave differently or make metadata available in a different way.
-* `instana-aws-fargate`: This is the production base image that we provide to customers to monitor Node.js Fargate tasks. The production image is published to `icr.io` (the public IBM Container Registry). This happens on our CI system, see `packages/serverless/ci/pipeline.yaml`. The [CI pipeline](https://ci.instana.io/teams/nodejs/pipelines/serverless-in-process-collectors:main/jobs/aws-fargate-nodejs-container-image-layer) uses the Dockerfile and package.json file in that folder, but not the build scripts `build.sh` or `build-and-push.sh`. These scripts are used to build variants of this image locally and to push them to a AWS ECR container registry, which is very usefull for testing. Available scripts:
+* `inspector`: A simple Node.js application that inspects its surroundings. It prints environment variables and queries the ECS metadata server, then prints the resulting HTTP responses. When running this in a Fargate task, it provides valuable insight into what a live Fargate application has available. This mostly holds historical value, as `@instana/aws-fargate` is pretty feature complete. However, it could become useful again if new AWS Fargate runtime versions behave differently or make metadata available in a different way.
```

Comment:
Question: As you went through the process of building fargate images for CJS & ESM. Are the steps in this document complete?

Reply:
All the necessary information is available in the readme file. To enhance comprehension, I'll make slight modifications to the documentation.